Windows: make docker-init fails with CreateProcess env bash error

## Summary
On Windows, `make docker-init` fails before running `scripts/docker.sh` due to shell launcher incompatibility.

## Reproduction
1. On Windows (PowerShell), run:
   ```bash
   make docker-init
   ```
2. Observe error:
   ```text
   process_begin: CreateProcess(NULL, env bash C:\Users\...\scripts\docker.sh init, ...) failed.
   make (e=2): The system cannot find the file specified.
   make: *** [Makefile:133: docker-init] Error 2
   ```

## Expected behavior
`make docker-init` should execute `scripts/docker.sh init` and continue to Docker environment checks.

## Root cause hypothesis
GNU Make on Windows can fail when recipes invoke shell scripts directly (`./scripts/*.sh`), because resolution through `env bash` depends on Unix-style tooling being available on PATH in a compatible way.

## Suggested fix
Call scripts explicitly via `bash` in Make recipes, e.g.:
- `bash ./scripts/docker.sh init`
- `bash ./scripts/docker.sh start`
- `bash ./scripts/docker.sh stop`

(And similarly for other `.sh` targets in root `Makefile`.)

## Notes
After replacing direct script invocations with explicit `bash`, `make docker-init` runs successfully on Windows and proceeds to normal Docker daemon checks.

--- a/Makefile
+++ b/Makefile
@@ -8,8 +8,11 @@ BASH ?= bash
 ifeq ($(OS),Windows_NT)
     SHELL := cmd.exe
     PYTHON ?= python
+    # Run repo shell scripts through Git Bash when Make is launched from cmd.exe / PowerShell.
+    RUN_WITH_GIT_BASH = call scripts\run-with-git-bash.cmd
 else
     PYTHON ?= python3
+    RUN_WITH_GIT_BASH =
 endif
 
 help:
@@ -48,7 +51,7 @@ config:
 	@$(PYTHON) ./scripts/configure.py
 
 config-upgrade:
-	@./scripts/config-upgrade.sh
+	@$(RUN_WITH_GIT_BASH) ./scripts/config-upgrade.sh
 
 # Check required tools
 check:
@@ -106,78 +109,46 @@ setup-sandbox:
 # Start all services in development mode (with hot-reloading)
 dev:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --dev
-else
-	@./scripts/serve.sh --dev
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --dev
 
 # Start all services in dev + Gateway mode (experimental: agent runtime embedded in Gateway)
 dev-pro:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --dev --gateway
-else
-	@./scripts/serve.sh --dev --gateway
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --dev --gateway
 
 # Start all services in production mode (with optimizations)
 start:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --prod
-else
-	@./scripts/serve.sh --prod
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --prod
 
 # Start all services in prod + Gateway mode (experimental)
 start-pro:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --prod --gateway
-else
-	@./scripts/serve.sh --prod --gateway
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --prod --gateway
 
 # Start all services in daemon mode (background)
 dev-daemon:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --dev --daemon
-else
-	@./scripts/serve.sh --dev --daemon
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --dev --daemon
 
 # Start daemon + Gateway mode (experimental)
 dev-daemon-pro:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --dev --gateway --daemon
-else
-	@./scripts/serve.sh --dev --gateway --daemon
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --dev --gateway --daemon
 
 # Start prod services in daemon mode (background)
 start-daemon:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --prod --daemon
-else
-	@./scripts/serve.sh --prod --daemon
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --prod --daemon
 
 # Start prod daemon + Gateway mode (experimental)
 start-daemon-pro:
 	@$(PYTHON) ./scripts/check.py
-ifeq ($(OS),Windows_NT)
-	@call scripts\run-with-git-bash.cmd ./scripts/serve.sh --prod --gateway --daemon
-else
-	@./scripts/serve.sh --prod --gateway --daemon
-endif
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --prod --gateway --daemon
 
 # Stop all services
 stop:
-	@./scripts/serve.sh --stop
+	@$(RUN_WITH_GIT_BASH) ./scripts/serve.sh --stop
 
 # Clean up
 clean: stop
@@ -193,29 +164,29 @@ clean: stop
 
 # Initialize Docker containers and install dependencies
 docker-init:
-	@./scripts/docker.sh init
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh init
 
 # Start Docker development environment
 docker-start:
-	@./scripts/docker.sh start
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh start
 
 # Start Docker in Gateway mode (experimental)
 docker-start-pro:
-	@./scripts/docker.sh start --gateway
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh start --gateway
 
 # Stop Docker development environment
 docker-stop:
-	@./scripts/docker.sh stop
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh stop
 
 # View Docker development logs
 docker-logs:
-	@./scripts/docker.sh logs
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh logs
 
 # View Docker development logs
 docker-logs-frontend:
-	@./scripts/docker.sh logs --frontend
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh logs --frontend
 docker-logs-gateway:
-	@./scripts/docker.sh logs --gateway
+	@$(RUN_WITH_GIT_BASH) ./scripts/docker.sh logs --gateway
 
 # ==========================================
 # Production Docker Commands
@@ -223,12 +194,12 @@ docker-logs-gateway:
 
 # Build and start production services
 up:
-	@./scripts/deploy.sh
+	@$(RUN_WITH_GIT_BASH) ./scripts/deploy.sh
 
 # Build and start production services in Gateway mode
 up-pro:
-	@./scripts/deploy.sh --gateway
+	@$(RUN_WITH_GIT_BASH) ./scripts/deploy.sh --gateway
 
 # Stop and remove production containers
 down:
-	@./scripts/deploy.sh down
+	@$(RUN_WITH_GIT_BASH) ./scripts/deploy.sh down

